src/Symfony/Component/Mailer/CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ CHANGELOG
    you now need to use the `IncompleteDsnTestTrait`.
 
  * Make `TransportFactoryTestCase` compatible with PHPUnit 10+
- * Support unicode email addresses such as "dømi@dømi.fo"
+ * Support unicode email addresses such as "dømi@example.org"
```

Comment:
What's the reason for this change?

Reply:
I think it's a good practice to use a domain that is reserved by IANA for documentation purposes, instead of a domain that could potentially be registered for malicious use in the future.
I'm aware this makes the example a little less explicit (no unicode in the domain part anymore), but I think it shows the meaning enough already.

Reply:
ah, works for me 👍

Reply:
I would use `dømi.example` as per RFC 2606 which recommends using the `.example` TLD for documentation.